modules/__tests__/Link-test.js

```diff
@@ -225,6 +225,43 @@ describe('A <Link>', function () {
     });
   });
 
+  describe('when route changes', function() {
+    it('changes active state', function(done) {
+      var LinkWrapper = React.createClass({
+        render() {
```

Comment:
How about we put `shouldComponentUpdate() { return false }` on it just to be sure it works?